docs/installation/debian.md

```diff
@@ -75,6 +75,11 @@ from the new repository:
 
             deb https://apt.dockerproject.org/repo debian-stretch main
 
+> **Note**: Docker does not provide packages for all architectures. To install docker on
+> a multi-architecture system, add an `[arch=...]` clause to the entry. Refer to the
+> [Debian Multiarch wiki](https://wiki.debian.org/Multiarch/HOWTO#Setting_up_apt_sources)
+> for details.
+
```

Comment:
Fixed; thanks, that is neat! Note that I added 2 spaces instead of 4, as it appears that lines it up with "The possible entries are:" instead of with each entry, which makes more sense and looks better to me. I'll change it to 4 spaces if you prefer that.